### DISCUSSION TITLE
Add Brown Color to Palette

### DISCUSSION THREAD

**Discussion by skylerknight** [tailwindlabs/tailwindcss#4245] [Ideas]
Hi,

It would be great if a true brown color could be added to the color palette. I know it's hard to get a decent color brown, but something would be better than nothing for sure. I noticed that even the official Tailwind site is using a background of "bg-orange-500" and then and inner div with "bg-black bg-opacity-75" to create a true brown color.

Thanks for your consideration : )

**Comment by RXminuS** [comment]
Fun fact...there is no such thing as brown in RGB made by light. There's only orange in a darker shade/luminosity compared to the context. That's why they do the `bg-orange-500` and opacity trick.

**Reply by RXminuS** [reply]
For those interested...this was pretty fun explanation by TechnologyConnections. https://www.youtube.com/watch?v=wh4aWZRtTwU&t=931s

**Reply by JordanMalan** [reply]
It would be nice if this was at least a part of colors, I mean if rose is a variant of red, brown could be a variant of orange :)

**Reply by MeikyuuTrader** [reply]
Although that's cool, it doesn't really matter, brown is just a standard colour and having to use tricks to achieve it is annoying. 

I guess the only option is to add a brown hex colour.

**Reply by JordanMalan** [reply]
Kinda... brown isn't really a standalone color... it's a shade of orange. 
You may remember roygbiv.

But I think if tailwind can have 5 different collections of grey then yeah, brown/tan should be on the pallet

**Comment by jonathanstanley** [comment]
Here is a brown option: https://davidpiesse.github.io/tailwind-md-colours/

just add the [source colors](https://gist.github.com/davidpiesse/74f5eaa23eb405e61b58cfe535d9907c#file-tailwind_md_all_colours-js-L255-L265) to [tailwind.config.js/theme/colors](https://tailwindcss.com/docs/customizing-colors#using-the-default-colors)
